Fatal error: exception Failure("duplicate local a in main")